### PR TITLE
Rename the gem to rails-bigint-primarykey

### PR DESCRIPTION
**Pull request by rafaelfranca**
This will make possible to us to release it in rubygems.org and it will make explicit this is a rewrite of the original gem.

After merging this I'll rename the repository and ask GitHub support to remove the fork link.

### CODE REVIEW

File: lib/bigint_primarykey.rb [hunk 1]
```diff
@@ -85,7 +85,7 @@ def install_patches!(adapter)
         pk_module = MysqlBigintPrimaryKey
```
Comment (lugray) [inline lib/bigint_primarykey.rb]: Not strictly necessary, but I'd `s/pk_module/primary_key_module/` here and L81,L95

**Comment by lugray** [comment]
Another time, it would be helpful for review to have the rubocop applied changes in a separate commit.

**Comment by rafaelfranca** [comment]
> Another time, it would be helpful for review to have the rubocop applied changes in a separate commit.

Yup. Totally agree. Let me fix this.

**Comment by rafaelfranca** [comment]
Cleaned the changes to not include the rubocop fixes. Also fixed the comment

File: test/gemfiles/Gemfile-Rails-4.2 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-# Specify your gem's dependencies in rails-bigint-pk.gemspec
+# Specify your gem's dependencies in rails-bigint-primarykey.gemspec
```
Comment (lugray) [inline test/gemfiles/Gemfile-Rails-4.2:3]: This change is needed in `test/gemfiles/Gemfile-Rails-5.0` too.

File: Readme.md [hunk 3]
```diff
@@ -23,3 +29,6 @@ end
 # Doing this will not use 64bit ints
 add_column :my_table, :other_table_id, :int
 ```
+
+When upgrading to Rails 5.1 you still need this gem if your were using it before and your migrations still use the
+verion 5.0 or previous.
```
Comment (Edouard-chin) [inline Readme.md]: `verion` -> `version`

**Review by lugray — APPROVED** [review]

**Comment by Edouard-chin** [comment]
Educate me here, should a fork have it's own versioning ? Just curious about the `2.0.0` for the first version of a fork

**Comment by rafaelfranca** [comment]
> Educate me here, should a fork have it's own versioning?

It can, but in this case I prefer to keep the version schema the same as the original one. It is fine to jump versions, the first version of actioncable for is the 5.0.0.

**Pull request merged**